src/style-spec/reference/v8.json

```diff
@@ -2814,11 +2814,16 @@
         }
       },
       "image": {
-        "doc": "Returns an `image` type for use in `icon-image` and `*-pattern` entries. If set, the `image` argument will check that the requested image exists in the style and will return either the resolved image name or `null`, depending on whether or not the image is currently in the style. This validation process is synchronous and requires the image to have been added to the style before requesting it in the `image` argument.",
+        "doc": "Returns an `image` type for use in `icon-image`, `*-pattern` entries and as a section in `format` expression. If set, the `image` argument will check that the requested image exists in the style and will return either the resolved image name or `null`, depending on whether or not the image is currently in the style. This validation process is synchronous and requires the image to have been added to the style before requesting it in the `image` argument.",
         "group": "Types",
         "sdk-support": {
           "basic functionality": {
             "js": "1.4.0"
+          },
+          "supported in expression": {
+            "format": {
```

Comment:
Why do we need nesting here? Could we simplify this so that it just reads `"image in format expression"`?

Reply:
@kkaefer Doing that would require a unique section in each property that can be nested into another in a non-trivial way (for eg: with `format`). Do you see an automated way of building this into a support verification matrix?